test/FastTests/Server/Documents/Revisions/RevisionsTests.cs

```diff
@@ -163,14 +163,27 @@ public async Task CanGetRevisionsByIdAndTimeLazily()
                 }
                 using (var session = store.OpenSession())
                 {
-                    var revision                 =  session.Advanced.Revisions.Get<User>("users/1", DateTime.Now);
-                    var revisionLazily  =  session.Advanced.Revisions.Lazily.Get<User>("users/1",DateTime.UtcNow);
-                    var revisionLazily2  =  session.Advanced.Revisions.Lazily.Get<User>("users/2",DateTime.UtcNow);
+                    var time = DateTime.UtcNow;
```

Comment:
I think you actually fixed it here :D

Reply:
Did we change something under the covers recently since now we need to pass UTC time here?

Reply:
maybe we changed how the server treat/parse `DateTime.Now`? or the .net 6 changed that?

Reply:
it also depends on the local culture, no?

Reply:
the issue was https://issues.hibernatingrhinos.com/issue/RavenDB-17834